sympy/printing/llvmjitcode.py

```diff
@@ -0,0 +1,192 @@
+from __future__ import print_function, division
+
+'''
+Use llvmlite to create executable functions from Sympy expressions
+
+This module requires llvmlite (https://github.com/numba/llvmlite).
+'''
+
+import ctypes
+
+from sympy.external import import_module
+from sympy.printing.printer import Printer
+from sympy import S
+from sympy.utilities.decorator import doctest_depends_on
+
+llvmlite = import_module('llvmlite')
+if llvmlite:
+    ll = import_module('llvmlite.ir').ir
+    llvm = import_module('llvmlite.binding').binding
+    llvm.initialize()
+    llvm.initialize_native_target()
+    llvm.initialize_native_asmprinter()
+
+
+class LLVMJitPrinter(Printer):
+    '''Convert expressions to LLVM IR'''
+    def __init__(self, *args, **kwargs):
+        kwargs.pop('func_args')
+        super(LLVMJitPrinter, self).__init__(*args, **kwargs)
+        self.fp_type = ll.DoubleType()
+        self.int_type = ll.IntType(64)
+
+    def _print_Number(self, n, **kwargs):
+        return ll.Constant(self.fp_type, float(n))
+
+    def _print_Integer(self, expr):
+        return ll.Constant(self.fp_type, float(expr.p))
```

Comment:
Ok, it would be worth adding that to the docstring for users to know. I'm also curious what happens if there are SymPy types that don't convert. Is it possible to give a nice error message to users?